src/text_shaper/open_shaper.cpp

```diff
@@ -346,12 +346,17 @@ namespace // {{{ helper
 
         if (_fd.spacing != font_spacing::proportional)
         {
-            if (_fd.familyName != "monospace")
-                FcPatternAddString(pat.get(), FC_FAMILY, (FcChar8 const*) "monospace");
 #ifdef _WIN32
             // On Windows FontConfig can't find "monospace". We need to use "Consolas" instead.
-            else
+            if (_fd.familyName == "monospace")
                 FcPatternAddString(pat.get(), FC_FAMILY, (FcChar8 const*)"Consolas");
+#elif __APPLE__
+            // Same for macOS, we use "Menlo" for "monospace".
+            if (_fd.familyName == "monospace")
+                FcPatternAddString(pat.get(), FC_FAMILY, (FcChar8 const*)"Menlo");
+#else
+            if (_fd.familyName != "monospace")
```

Comment:
I also don’t quite understand this. This is the original logic without all my fixes. Is this line force the user to use “monospace” provided by system? What if the user choose a different font? Will this override that? I didn’t really try. Just curious.

Reply:
To be fair, fontconfig documentation wasn't really that insightful to me. And while having had a look at quite some other source places I came up with this one. So I think it is "okay" to merge what you think works better especiall on Win32 for now. and we can always reiterate and improve. :)